### ISSUE TITLE
Export error when project directory contains `-r`

### ISSUE THREAD

**Issue by chrodriguez** [jsonresume/resume-cli#384]
I've got this error the first time I've tested the library. I' thought I was doing something wrong but, then realized the problem was really subtle. When exporting and parent directory name contains substring `-r` or `--rename`, export action fails.
This is because arguments are parsed using  `indexOf('-r')` and that function matches for example my use case:  a directory named  `my-resume/`.
Error can be reproduced following this actions:

```bash
mkdir /tmp/my-resume
cd /tmp/my-resume
npm init -f && npm i resume-cli
./node_modules/.bin/resume init
./node_modules/.bin/resume export test.html
cat test.html # <= This HTML/PDF is empty
```
And as example shows, exported file will be empty because `resume.json` is not found.

**Comment by chrodriguez** [comment]
I've wrote a fix, but seems #362 just do the same fix I've done

**Comment by thomasdavis** [comment]
Yeah, I might close this issue in favor of https://github.com/jsonresume/resume-cli/pull/362 

And as discussed in that issue, it might be worth correctly passing arguments.
